test/e2e/serial/tests/configuration.go

```diff
@@ -693,9 +697,195 @@ var _ = Describe("[serial][disruptive] numaresources configuration management",
 				return kcCmNamesCur
 			}).WithContext(ctx).WithTimeout(5 * time.Minute).WithPolling(10 * time.Second).Should(Equal(kcCmNamesPre))
 		})
+
+		It("[reboot_required][slow] should be able to correctly identify topology manager policy without scheduler restarting", Label("reboot_required", "slow"), func() {
```

Comment:
let's make sure this is `[tagged]` and `Label()`ed `disruptive`, as it restarts the kubelet

Reply:
All the tests in this file is under 1 Describe and it already has disruptive should I put another it also under It() or its considered a duplicate?

Reply:
no, if the parent `Describe` is already tagged and labeled `disruptive` then we're good.